cypress/e2e/ApplicationNewPage.e2e.js

```diff
@@ -35,303 +35,303 @@ describe('ApplicationNewPage', () => {
-    cy.get('#household_members_0_date_of_birth_year').type(HOUSEHOLD_MEMBER_DOB_YEAR)
-
-    if (usingFixtures()) {
-      cy.intercept('GET', 'api/v1/short-form/**', { fixture: 'shortFormGet2.json' }).as(
-        'shortFormGet'
-      )
-    } else {
-      cy.intercept('GET', 'api/v1/short-form/**').as('shortFormGet')
-    }
-
-    // Save the application and verify there are no form errors
-    cy.get('.save-btn').click()
-    cy.wait('@shortFormPost')
-    cy.wait('@shortFormGet')
-
-    cy.get('.alert-box').should('not.exist')
-
-    cy.get('.application-details')
-
-    cy.url().should('match', /\/applications\/.*\?showAddBtn=true/)
-
-    // We get all the application attributes values including household members
-    cy.contains(TRUNCATED_FIRST_NAME).should('exist')
-    cy.contains(TRUNCATED_LAST_NAME).should('exist')
-    cy.contains(DATE_OF_BIRTH).should('exist')
-    cy.contains(HOUSEHOLD_MEMBER_FIRST_NAME).should('exist')
-    cy.contains(HOUSEHOLD_MEMBER_LAST_NAME).should('exist')
-    cy.contains(HOUSEHOLD_MEMBER_DATE_OF_BIRTH).should('exist')
-  })
-
-  it('should create a new application with live/work preference successfully', () => {
-    cy.visit('http://localhost:3000/')
-    cy.login()
-    cy.visit(`/listings/${NON_LEASE_UP_LISTING_ID}/applications/new`)
-
-    // Fill out primary applicant required fields
-    cy.fillOutRequiredFields()
-
-    cy.get('#add-preference-button').click()
-
-    // Fill out live/work preference required fields
-    // Select the live/work preference
-    cy.get('#select-paper-preference-0').select('a0l0P00001Lx8XeQAJ') // Live/Work dropdown value
-
-    // Verify that expected fields are required.
-    cy.get('.save-btn').click()
-    cy.contains('Household Member with Proof is required').should('exist')
-    cy.contains('Individual Preference is required').should('exist')
-    cy.contains('Type of Proof is required').should('exist')
-
-    // Select the household member
-    cy.get('#form-preferences\\.0\\.naturalKey')
-      .select(`${TRUNCATED_FIRST_NAME},${TRUNCATED_LAST_NAME},${DOB_YEAR}-${DOB_MONTH}-${DOB_DAY}`)
-      .blur()
-
-    // Check ability to switch between the 2
-    // options with the type of proof updating accordingly
-
-    // Select work vs live
-    cy.get('#form-preferences\\.0\\.individual_preference').select('Work in SF').blur()
-
-    // Select type of work proof
-    cy.get('#form-preferences\\.0\\.type_of_proof').select('Letter from employer').blur()
-
-    // Select live vs work
-    cy.get('#form-preferences\\.0\\.individual_preference').select('Live in SF').blur()
-
-    // Verify that preference proof has been unset and is validated.
-    cy.get('.save-btn').click()
-    cy.contains('Type of Proof is required').should('exist')
-
-    // Select type of proof
-    cy.get('#form-preferences\\.0\\.type_of_proof').select('Telephone bill').blur()
+  // it('should fail if required fields are missing, and create application if not', () => {
```

Comment:
suggestion: can you add to the ticket or this PR why we are commenting out tests?

Reply:
I was just doing that to make it faster to run locally! sorry I'll uncomment these